src/main/webapp/app/entities/feedback.model.ts

```diff
@@ -18,6 +18,8 @@ export enum FeedbackType {
     AUTOMATIC_ADAPTED = 'AUTOMATIC_ADAPTED',
 }
 
+export const STATIC_CODE_ANALYSIS_FEEDBACK_IDENTIFIER = 'SCAFeedbackIdentifier:';
```

Comment:
add a note that this must be equal to the constant on the server (including the link to the corresponding file)